app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -121,7 +121,7 @@ echo "#"
 
 <% if plugin_present?('katello') && activation_keys.present? -%>
 register_katello_host(){
-    UUID=$(subscription-manager identity | head -1 | awk '{print $3}')
+    UUID=$(subscription-manager identity | sed -ne 's,.*\([[:xdigit:]]\{8\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{12\}\).*,\1,p; T; q')
```

Comment:
```suggestion
    UUID=$(subscription-manager identity | grep -m 1 -o '\([[:xdigit:]]\{8\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{12\}\)')
```

Ok this two versions work on RHEL I care. Choose which one you want.

Reply:
Hi,
thanks for the PR. I just wanted to ask, what's the reason for these changes? Is it solving some issue with getting the UUID?

Reply:
I explained the reason in the associated ticket. Tltr: when I debugged an issue I saw this snippet broke if another locale is used so send a PR.

Reply:
```
export LANG=fr_FR;echo $(subscription-manager identity | grep -m 1 -o '\([[:xdigit:]]\{8\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{4\}-[[:xdigit:]]\{12\}\)')
```
Tested variant with the grep on Rhel 7, Rhel 8 and RHEL 9, all of them worked fine/
I'm voting +1 for the grep variant